README.md

```diff
@@ -0,0 +1,41 @@
+# DigiForest DRS
+
+DRS tooling developed for the DigiForest project
+
+Authors: Benoit Casseau, Matias Mattamala, Nived Chebrolu, Maurice Fallon
+
+## Package descriptions
+
+- `digiforest_analysis`: Python modules to extract tree data
+- `digiforest_analysis_ros`: ROS wrappers
+- `digiforest_slam_logger`: Tool to export a state estimate solution in the SLAM format
+
+## Development
+
+### Setup
+
+Use a virtual environment (named `env` in the example):
+
+```sh
+python3 -m venv env --system-site-packages 
```

Comment:
Why the flag ```--system-site-packages```? Isn't it just better to keep a copy of the libraries with a specific version number for this project separately?

Reply:
If we don't use the system packages we don't have ROS. I guess we manually install rospy in the environment too